tests/lib/k8s/test_builders.py

```diff
@@ -129,6 +129,20 @@ def test_get_anti_affinity_zone_enabled_for_process(self, parse_yaml_mock):
         )
         self.assertEqual(anti_affinity_zone, k8s_fixtures.anti_affinity_zone)
 
+    def test_get_image_name_branch(self):
+        """Returns the URL of the docker image if tag and branch are provided"""
+        app_config = {"app": "app", "registry": {"organization": "my-organization"}}
+        image_url = k8s_builders.get_image_name(
+            app_config, {"branch": "feature/api", "tag": "1.0.0-sha-a2b3c4"}
```

Comment:
You don't really need `tag` anymore, but it is still good to assert that `branch` is taken first if present